scapy/route6.py

```diff
@@ -30,8 +30,8 @@
 class Route6:
 
     def __init__(self):
-        self.invalidate_cache()
```

Comment:
Why are you changing this ?

Reply:
Only to have consistency between route.py and route6.py. In some previous commits I had modified stuff here, and i just let this change so we have the same order of execution in the two files.

I can remove it if needed

Reply:
Got it!